drift_dev/lib/drift_dev.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:custom_lint_builder/custom_lint_builder.dart';
+import 'package:drift_dev/src/lints/custom_lint_plugin.dart';
+
+/// This function is automaticly recognized by custom_lint to include this drift_dev package as a linter
+PluginBase createPlugin() {
```

Comment:
Not a huge fan of this, is there a way to tell `custom_lint` to use a different import for a package providing linters (my preferred library would be `lib/integrations/custom_lint.dart`).

But if that's not possible so be it, I can file an issue on `custom_lint` and we can leave this as-is until that gets resolved.